src/main/routes.ts

```diff
@@ -167,6 +171,10 @@ export class Routes {
       );
 
       app.get(`${MIAM_START}/:caseId`, errorHandler(new MIAMGetController(step.view, step.generateContent).get));
+      app.get(
```

Comment:
Why is the route not registered in steps and directly in route files?

Reply:
This is followed with similar approach as with other controllers used in.